src/components/Filters/components/FilterSelect/FilterSelect.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import {Button, Icon, Popup, Sheet, useMobile} from '@gravity-ui/uikit';
+import {Plus} from '@gravity-ui/icons';
+
+import {block} from 'src/components/utils/cn';
+
+import {FilterListContainer} from '../FilterListContainer/FilterListContainer';
+
+import i18n from './i18n';
+
+import './FilterSelect.scss';
+
+const b = block('filter-select');
+
+interface FilterSelectItem {
+    value: string;
+    label: string;
+    description?: string;
+}
+
+interface FilterSelectProps {
+    className?: string;
+
+    text?: string;
+
+    options: FilterSelectItem[];
+    values: string[];
+
+    onItemClick: (value: string) => void;
+
+    defaultOpen?: boolean;
+    showButtonText?: boolean;
+}
+
+export function FilterSelect(props: FilterSelectProps) {
```

Comment:
Why we do not use [Select](https://github.com/gravity-ui/uikit/tree/main/src/components/Select) here?

Reply:
I believe it's for consistency, filters controls and filter select control should have same styling